issues-workflow.md

```diff
@@ -40,12 +41,15 @@ There are five different stages of the triage:
 | Validity | Asses the validity of the issue (whether it is taken for the further triage and proper classification). | invalid, duplicate, wontfix, question |
 | Kind | Differentiate whether the related issue is a new feature or a bug. | enhancement, bug |
 | Help | Identify issues that do not have high priority and can be taken by the community. | help wanted, good first issue|
-| Priority | Prioritize issues in the general Kyma backlog to select those which are the most critical and should be taken as first. | critical |
+| Security | Specify CVSSv3 security rating https://www.first.org/cvss/calculator/3.0 with support of security team | security/{SCORE} |
```

Comment:
`Specify the [CVSSv3](https://www.first.org/cvss/calculator/3.0) security rating with the support of the security team.`